assets/scripts/app/routing.js

```diff
@@ -68,5 +73,5 @@ export function goCopyLastStreet () {
 }
 
 export function goSignIn () {
-  window.location.href = '/' + URL_SIGN_IN_REDIRECT
+  window.location.href = '/' + TWITTER_URL_SIGN_IN_REDIRECT
```

Comment:
Can this be replaced by extracting the WebAuth from `MenuBar.jsx` so that `goSignIn()` runs the authentication from Auth0?